loadtest/control/simulcontroller/actions.go

```diff
@@ -80,6 +82,14 @@ func (c *SimulController) reload(full bool) control.UserActionResponse {
 		}
 	}
 
+	defer func() {
+		elapsed := time.Since(start).Seconds()
+		err := c.user.ObserveClientMetric(model.ClientPageLoadDuration, elapsed/1000)
```

Comment:
Why are we dividing? The point of passing a float is that we can use seconds as unit as expected by most standard metrics.

Reply:
Ops my bad, I directly copied the suggestion, it should've been `Milliseconds`, not `Seconds`.